### PR TITLE
inject asset packages in assets helper service

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #14368 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php [hunk 1]
```diff
@@ -558,6 +558,10 @@ private function registerTemplatingConfiguration(array $config, $ide, ContainerB
                 'Symfony\\Bundle\\FrameworkBundle\\Templating\\Loader\\FilesystemLoader',
             ));
         }
+
+        if ($container->hasDefinition('assets.packages')) {
+            $container->getDefinition('templating.helper.assets')->replaceArgument(0, new Reference('assets.packages'));
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: the helper service should be removed otherwise

Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: done

Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php]: Though I didn't find circumstances where this ever will happen. If anyone can think of it, we could add a test covering it.

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**